SPECS/kernel-headers/kernel-headers.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Linux API header files
 Name:           kernel-headers
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
I think we also need to update `kernel-hyperv` the same way.

Reply:
Thanks, fixed